README.md

```diff
@@ -1 +1,179 @@
-# react-metrics-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
```

Comment:
- [ ] The [video presentation](https://www.loom.com/share/5bee8d89db6343599ccb7c8192f51ee5) lacks highlights of a piece of cod that you finds interesting or are proud of making. Kindly redo this video including a show of a piece of code that you are proud of.

Reply:
Yes, I have uploaded another video. Thank you